src/Sulu/Bundle/AdminBundle/Resources/public/js/components/header/main.js

```diff
@@ -501,13 +501,15 @@ define([], function() {
         startLanguageChanger: function() {
             if (!!this.options.toolbarLanguageChanger) {
                 var $element = this.sandbox.dom.createElement(this.sandbox.util.template(templates.languageChanger)({
-                        title: this.options.toolbarLanguageChanger.preSelected || this.sandbox.sulu.user.locale
+                            title: this.options.toolbarLanguageChanger.preSelected
+                                || this.sandbox.sulu.getDefaultContentLocale()
```

Comment:
Isn't that the exact error we had before? This method is simply returning the first localization in the system again, isn't it?